backend/api/utils.py

```diff
@@ -80,7 +80,7 @@ def get(self):
         for a in module_attrbs:
             attrb = {}
             attrb['id'] = a.__name__
-            attrb['options'] = [{ 'text' : elem.description if hasattr(elem, 'description') else elem.name, 'id' : str(elem.value) } \
+            attrb['options'] = [{ 'text' : elem.description if hasattr(elem, 'description') else elem.name, 'id' : elem.value } \

```

Comment:
@chernyee Please approve this change. I need number type at my side. Otherwise I have to do double conversion.